src/store/user.vuex.ts

```diff
@@ -89,8 +89,8 @@ export default function createUserStore($http: AxiosInstance) {
         const userDataResponse = (await axios.get(`/get/?type=my_user&uid=${uid}`)).data.data;
         this.userDetails = userDataResponse.user;
         this.hasLabAccess = Boolean(
-          Number(this.userDetails.ten_tools_level) >= 8 ||
-            Number(this.userDetails.is_lab_member_legacy),
+          Number((this.userDetails as UserData).ten_tools_level) >= 8
+            || Number((this.userDetails as UserData).is_lab_member_legacy),
```

Comment:
Changing line 90 to `this.userDetails = userDataResponse.user as UserData;` removes the need for typing here.

Reply:
Could we have done `public userDetails: UserData?` instead? (I'm pretty new to typescript so not sure)